src/plugins/expression_metric/public/expression_renderers/metric_renderer.tsx

```diff
@@ -46,22 +43,27 @@ export const getMetricRenderer =
       config: MetricRendererConfig,
       handlers: IInterpreterRenderHandlers
     ) => {
+      const { MetricComponent } = await import('../components/metric_component');
       handlers.onDestroy(() => {
         unmountComponentAtNode(domNode);
       });
 
       render(
-        <KibanaThemeProvider theme$={theme$}>
-          <MetricComponent
-            label={config.label}
-            labelFont={config.labelFont ? (config.labelFont.spec as CSSProperties) : {}}
-            metric={config.metric}
-            metricFont={config.metricFont ? (config.metricFont.spec as CSSProperties) : {}}
-            metricFormat={config.metricFormat}
-          />
-        </KibanaThemeProvider>,
+        <EuiErrorBoundary>
```

Comment:
Since we no longer load the component using `withSuspense` we wrap the component in `EuiErrorBoundary`.